resources/definitions/os_discovery/arubaos-cx.yaml

```diff
@@ -36,6 +49,49 @@ modules:
                     descr: 'Power Supply {{ $arubaWiredPSUName }}'
                     value: arubaWiredPSUInstantaneousPower
                     high_limit: arubaWiredPSUMaximumPower
+                -
+                    oid: arubaWiredPmXcvrLaneDomTable
+                    num_oid: '.1.3.6.1.4.1.47196.4.1.1.3.27.1.2.2.1.7.{{ $index }}'
+                    value: arubaWiredPmXcvrLaneDomTxPower
+                    descr: "{{ $subindex0 }} TX Power"
+                    divisor: 1000
+                    group: transceiver
+                    index: 'txpower-{{ $subindex0 }}'
+                    entPhysicalIndex: '{{ $subindex0 }}'
+                    entPhysicalIndex_measured: ports
+                -
+                    oid: arubaWiredPmXcvrLaneDomTable
+                    num_oid: '.1.3.6.1.4.1.47196.4.1.1.3.27.1.2.2.1.12.{{ $index }}'
+                    value: arubaWiredPmXcvrLaneDomRxPower
+                    descr: "{{ $subindex0 }} RX Power"
```

Comment:
i don't yet test with transceiver using multiple lane and i don't found how to get name of interface (it is not available on index)

Reply:
You can pull in IF-MIB::ifName and use that.